docs/docs/how-tos/nebari-aws.md

```diff
@@ -115,7 +115,7 @@ environment variables have been properly set. It is time to initialize and deplo
    nebari init --guided-init
    ```
 
-   ![A representation of the output generated when Nebari init guided-init command is executed.](/img/how-tos/nebari-aws.png)
+   ![A representation of the output generated when Nebari init guided-init command is executed. ](/img/how-tos/nebari-aws.png)
```

Comment:
I'll work on this.

Reply:
@pavithraes It might be better to add more description to the text above instead of the alt text on a screenshot.  I'm also wondering if the screenshot could be pasted in as a code block instead of an image. What do you think?